assets/js/componentDataTables.js

```diff
@@ -49,7 +42,18 @@ class ComponentDataTable {
             "<'row'<'col-sm-5'i><'col-sm-7'p>>",
             buttons: [{
                 name: "uploadDeja",
-                text: '<i class=" fa fa-cloud-upload"></i> Upload Components'
+                text: '<i class=" fa fa-cloud-upload"></i> Upload Components',
+                action: ( e, dt, node, config ) => {
+                    this.aboutCodeDB.findAllComponents( {} )
+                        .then((components) => {
+                            if (components.length > 0) {
+                                $('#componentExportModal').modal('show');
+                            }
+                            else {
```

Comment:
Will do @jdaguil .  (I'll need to [do the same](https://github.com/nexB/aboutcode-manager/blob/3b771de637f4c167ae48fcb83725f31fa897b465/assets/js/renderer.js#L657) for PR #135  but will wait in case you have other comments.)